index.html

```diff
@@ -480,13 +480,29 @@
-
-                <p>A fundamental question that must be answered is how does a <a>user agent</a> get hold of the manifest. Because the publication patterns of WPs can be different depending on publishers, authors, etc., there should be different ways of accessing the manifests: it can be referred to via a <code>link</code> element in an HTML file, can be in an agreed-upon-position within a package, or can be conveyed through a <code>LINK</code> header of an <code>HTTP(S)</code> request. The user agent gets hold of the <a>manifest</a> by following a hierarchy of these different possibilities; a separate <a href="#simple-manifest-access">section</a> in the appendix provides more details. (Note that accessing a manifest also has some security issues; the <a href="https://w3c.github.io/manifest/#obtaining">“Obtaining a manifest”</a> of the “Web App Manifest”&nbsp;[[web-manifest]] document provides some further details that could be adopted, too.)</p>
+                <p>A <a>Web Publication</a> is a whole that may consist of many parts. We need a way to describe this whole, a place to store information about the publication and its components. We will use the term <a>manifest</a>, in the sense of a document describing the contents of a ship. A <a>Manifest</a> may include:</p>
+                
+                <ul>
+                <li>Publication metadata, directly or by reference.</li>
+                <li>A list of publication resources, and information about them.</li>
+                <li>The default sequencing of primary publication components.</li>
+                <li>References to tables of contents, glossaries, and other navigational or pedagogical aids.</li>
+                <li>References to alternative manifestations of the publication.</li>
+                <li>Hints to the user agent about how to present the publication.</li>
+                
+                </ul>
+                
+                
+                
+                
+                 <p>The “Web App Manifest”&nbsp;[[web-manifest]], currently being developed at W3C, is one example of a technology that could define a PWP manifest format. Although the current&nbsp;[[web-manifest]] draft is geared towards Web Applications, it also includes extension points to add information necessary for PWPs. Some of these extra manifest members have already been explored in the <a href="https://github.com/dauwhe/epub31-bff">“Browser Friendly Format”</a> work of the EPUB3.1 Working Group. Given the broad range of uses we envision for <a>Web Publications</a>, there may be several alternatives for serializing the manifest, including EPUB’s package file. </p>
+                 
```

Comment:
Do we want to refer to EPUB at this point? Maybe the last remark could go. (It may raise too many eyebrows...)